verify.go

```diff
@@ -190,7 +202,7 @@ func VerifyRaw(input []byte, vc VerifyConfig) ([]byte, error) {
 			continue
 		}
 
-		if !key.IsPublic() {
+		if !vc.ExpectSymmetrical && !key.IsPublic() {
```

Comment:
should this also check for the inverse?  eg, if `vc.ExpectSymmetrical` is true, don't allow public keys?

Reply:
Added in 45aa7b6